src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Rijndael.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.Cryptography;
+using System.ComponentModel;
+
+namespace System.Security.Cryptography
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract class Rijndael : SymmetricAlgorithm
+    {
+        public static Rijndael Create()
+        {
+            return new RijndaelImplementation();
+        }
+
+        protected Rijndael()
+        {
+            LegalBlockSizesValue = s_legalBlockSizes.CloneKeySizesArray();
+            LegalKeySizesValue = s_legalKeySizes.CloneKeySizesArray();
+            KeySizeValue = 256;
+            BlockSizeValue = 128;
+            ModeValue = CipherMode.CBC;
```

Comment:
Nit: In NetFx's Rijndael it doesn't set ModeValue, leaving it up to the default from SymmetricAlgorithm.

Doesn't matter much; but pointing it out as a difference.